CHANGELOG.md

```diff
@@ -1,5 +1,3 @@
-# Changelog
```

Comment:
This is removed because all future additions to the changelog are pushed onto the beginning.